### PR TITLE
benrubson/xattr

### PR DESCRIPTION
**Pull request by rfjakob**
This is a rebase of https://github.com/vgough/encfs/pull/342 to current master
with slight modifications.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,6 +30,8 @@ before_script:
 
 script:
   - if [ "${COVERITY_SCAN_BRANCH}" != 1 ]; then make && make check ; fi
+  - cd ..
+  - ./test.sh
```
Comment (benrubson) [inline .travis.yml]: `make check` above already run tests :)

**Comment by rfjakob** [comment]
@benrubson Slightly modified and got rid of the error messages. You ok with this version?

**Comment by benrubson** [comment]
Sounds perfect to me !
Thank you Jakob for your work !

**Comment by benrubson** [comment]
Thank U !
#341 can now also be closed :)

**Pull request merged**